advanced_source/static_quantization_tutorial.py

```diff
@@ -348,7 +374,8 @@ def prepare_data_loaders(data_path):
     return data_loader, data_loader_test
 
 ######################################################################
-# Next, we'll load in the pre-trained MobileNetV2 model
+# Next, we'll load in the pre-trained MobileNetV2 model. Similarly to the data about, the file with the pre-trained
+# weights is stored at ``https://s3.amazonaws.com/pytorch-tutorial-assets/mobilenet_quantization.pth``:
```

Comment:
This is confusing to end users. Let us use the file from torchvision at: https://github.com/pytorch/vision/blob/master/torchvision/models/mobilenet.py#L9

Also, lets call this file something else: not mobilenet_quantization.pth, might be better to call it mobilenet_pretrained_float.pth'